app/views/donations/_status.html.erb

```diff
@@ -3,11 +3,7 @@
 <% elsif donation.declined? %>
   <%= t(".declined") %>
 <% else %>
-  <%= button_to(donation_confirmation_path(donation), method: :post) do %>
-    <%= t(".confirm") %>
-  <% end %>
+  <%= render("donations/confirm", donation: donation) %>
 
-  <%= button_to(donation_confirmation_path(donation), method: :delete) do %>
-    <%= t(".decline") %>
-  <% end %>
+  <%= render("donations/decline", donation: donation) %>
```

Comment:
I favor being explicit in this case as well. Future PRs will add different classes and such.

I favor the duplicated markup instead of passing in:

* the HTTP method
* the internationalized text / key
* any unique CSS classes
* data roles
* `[disabled]` flags
* etc.